Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -219,13 +219,20 @@ public class PlayerInput : MonoBehaviour
         /// <seealso cref="onDeviceRegained"/>
         public const string DeviceRegainedMessage = "OnDeviceRegained";
 
+        /// <summary>
+        /// Name of the message that is sent with <c>UnityEngine.Object.SendMessage</c> when the
+        /// controls used by a player are changed.
+        /// </summary>
+        /// <seealso cref="onControlsChanged"/>
+        public const string ControlsChangedMessage = "OnControlsChanged";
+
         /// <summary>
         /// Whether input is on the player is active.
         /// </summary>
         /// <value>If true, the player is receiving input.</value>
         /// <seealso cref="ActivateInput"/>
-        /// <seealso cref="PassivateInput"/>
-        public bool active => m_InputActive;
+        /// <seealso cref="DeactivateInput"/>
+        public bool inputIsActive => m_InputActive;
```

Comment:
Same here, when renaming public API, keep the old one as obsolete (at least until you make a breaking change, ie increase major version number).

Reply:
Fixed